usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -662,21 +729,14 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
             if award_count == 1:
                 file_d_award = award_queryset[0]
                 value_map_faba['award_id'] = file_d_award['id']
-                logger.info('individual award id mapped: {}'.format(file_d_award['id']))
+                del value_map_faba['award']  # remove a potentially bad reference to the related award
+                # logger.info('individual award id mapped: {}'.format(file_d_award['id']))
 
         # Still using the cpe|fyb regex compiled above for reverse
         load_data_into_model(award_financial_data, row, value_map=value_map_faba, save=True, reverse=reverse)
 
     awards_cache.clear()
 
```

Comment:
i added this check bulk because it is way faster than using django.

Reply:
The bulk operation didn't seem like it was limiting on only the rows in question, but instead after each submission, operate on the entire data set. This would inevitably run into the same issue as `update_description_fields` where the data set was too large for it to operate efficiently.